### DISCUSSION TITLE
Comparison with other codecs

### DISCUSSION THREAD

**Discussion by jay-ramani** [ultravideo/uvg266#24] [General]
Great work.

Would be good to see some comparison numbers on tools that help with other codecs (vvc vs. hevc vs. av1).

While there are external sites that provide [some information](https://www.winxdvd.com/video-transcoder/h266-vvc-vs-av1.htm), can the authors provide some competitive statistics?

Thanks and cheers.

**Comment by fador** [comment]
Thanks!
It would be interesting to do some comparisons, but uvg266 is nowhere near to be a complete VVC encoder, we are missing a lot of features that would allow us to achieve the reference encoder type of compression performance..

**Comment by fador** [comment]
There are some performance results in our recently published article about uvg266:
[https://dl.acm.org/doi/10.1145/3664647.3685525](https://dl.acm.org/doi/10.1145/3664647.3685525)

**Reply by jay-ramani** [reply]
I had already read this article, but there is no codec comparison numbers as such. Thanks for sharing though; might be useful for others.
